python/mxnet/symbol/symbol.py

```diff
@@ -1657,6 +1657,10 @@ def optimize_for(self, backend, args=None, aux=None, ctx=None,
 
         new_sym = Symbol(out)
 
+        # convert to numpy symbol if needed
+        if is_np_array():
+            new_sym = new_sym.as_np_ndarray()
+
```

Comment:
Should this be in the numpy `_Symbol` class? Ie overwrite the `optimize_for` method there to call `as_np_ndarray` after `super().optimize_for()` finishes?

Reply:
@leezu Yes, it can be there - I haven't noticed presence of `_Symbol` class.
Can you tell me if Sphinx will copy documentation from base class? I'm not sure if I should copy docstring there too

Reply:
> Can you tell me if Sphinx will copy documentation from base class?

They need to be specified specifically via `inherited-members` https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html